kv/kvclient/kvcoord: TestMuxRangeFeedDoesNotStallOnError failed

kv/kvclient/kvcoord.TestMuxRangeFeedDoesNotStallOnError [failed](https://tanzanite.cluster.engflow.com/invocations/default/1a77ae92-e72b-4284-9588-ffeca531d5f9?testReportRun=2&testReportShard=12&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZjbGllbnQva3Zjb29yZDprdmNvb3JkX3Rlc3Q=) on master @ [6eb6d4935baf41d60ba3667ed6b6c4ada25cdc26](https://github.com/cockroachdb/cockroach/commits/6eb6d4935baf41d60ba3667ed6b6c4ada25cdc26):


```
=== RUN   TestMuxRangeFeedDoesNotStallOnError
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestMuxRangeFeedDoesNotStallOnError1381550309
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:139: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
    dist_sender_rangefeed_test.go:377: condition failed to evaluate within 3m45s: from dist_sender_rangefeed_test.go:383: too many replicas: 3
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestMuxRangeFeedDoesNotStallOnError1381550309
--- FAIL: TestMuxRangeFeedDoesNotStallOnError (655.19s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=12</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMuxRangeFeedDoesNotStallOnError.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-45764

--- a/pkg/kv/kvclient/kvcoord/dist_sender_rangefeed_test.go
+++ b/pkg/kv/kvclient/kvcoord/dist_sender_rangefeed_test.go
@@ -25,6 +25,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/server"
 	"github.com/cockroachdb/cockroach/pkg/sql/catalog/desctestutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils"
+	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/testcluster"
 	"github.com/cockroachdb/cockroach/pkg/util"
@@ -309,6 +310,8 @@ func TestMuxRangeCatchupScanQuotaReleased(t *testing.T) {
 func TestMuxRangeFeedDoesNotStallOnError(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
+
+	skip.UnderDuress(t, "slow test")
 
 	ctx := context.Background()
 

